backend/matching-service/src/types/IUserQueue.ts

```diff
@@ -0,0 +1,9 @@
+import { Category, Proficiency } from '@repo/user-types'
+
+export interface IUserQueue {
```

Comment:
Should name this IUserQueueMessage for clarity

Reply:
Okay will do!